src/main/java/org/opentripplanner/graph_builder/linking/SimpleStreetSplitter.java

```diff
@@ -405,9 +405,7 @@ private SplitterVertex split (StreetEdge edge, LinearLocation ll, boolean tempor
                 edge);
         }
 
-        // make the edges
-        // TODO this is using the StreetEdge implementation of split, which will discard elevation information
-        // on edges that have it
+        // make the edges on edges that have it
```

Comment:
This comment already existed here earlier but I find it slightly hard to understand. Maybe it could be worded a bit better.